valgprogram.md

```diff
@@ -5,7 +5,8 @@
 
 ### Digitalt personvern
 
-Piratpartiet vil
+Piratpartiet vil:
+
```

Comment:
EOL-endring? Sikkert greit, men også fordelaktig å unngå hvis man kan. 😃

Reply:
fix'd in 82b9cace6aef3e86fd738a4d6370641adcf60538